backend/Recodo.DAL/Entities/Configuration/CommentReactionConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recodo.DAL.Entities.Configuration
+{
+    public class CommentReactionConfiguration : IEntityTypeConfiguration<CommentReaction>
+    {
+        public void Configure(EntityTypeBuilder<CommentReaction> builder)
+        {
+            builder.HasKey(p => p.Id);
+            builder.HasOne(p=>p.User).WithMany().HasForeignKey(p=>p.UserId);
```

Comment:
```suggestion
            builder.HasOne(p => p.User).WithMany().HasForeignKey(p=>p.UserId);
```
Cleanliness. Here and everywhere else